src/fdb5/api/FDB.h

```diff
@@ -71,72 +71,253 @@ class FDB {
 
     // -------------- Primary API functions ----------------------------
 
+    /** Archive binary data to a FDB.
+     *
+     * \param handle eckit::message::Message to data to archive
```

Comment:
The types can be omitted from the description. IMO this is just hard to keep in sync with the actual implementation